src/Dropdown/index.d.ts

```diff
@@ -0,0 +1 @@
+export { default as Dropdown, DropdownItemProps as DropdownProps } from './Dropdown';
```

Comment:
The `DropdownProps` type needs to be exported because it's used in ButtonDropdown.d.ts. I suppose you mean this?
```suggestion
export { default as Dropdown, DropdownProps } from './Dropdown';
```